provisioning/atat_provisioning.yaml

```diff
@@ -1,21 +1,48 @@
 openapi: 3.0.2
 info:
   description: >-
-    This is a DRAFT version of the API meant for provisioning resources into a JWCC Provider Cloud Environment. It will be
-    implemented and maintained by all participating CSPs.
+    This API is meant for provisioning resources in a JWCC Provider Cloud Environment.
+    The cloud products and services used in JWCC are provided by commercial cloud service providers (CSP).
+    This API will be implemented and maintained by all participating CSPs.
+
+    ### Organization of provisioned resources
+    
+    Provisioned resources under the purview of a mission owner are organized in a three-level hierarchy.
+
+    * Portfolio
+      * Application
+        * Environment
+
+    *Portfolios* consist of one or more applications. *Applications* consist of one or more environments.
+    *Environments* directly contain provisioned resources.
+
+    Access can be granted to *operators* at each level.
+
```

Comment:
Operators are specified generically as email/access level tuples. See '#/components/schemas/PortfolioOperatorRole' for an example. ATAT will request that individuals (uniquely identified by email address) be granted access to specific resources (Portfolios, Applications or Environments). It is expected that CSPs will manage account lifecycle and authorize Operators according to these requests. 

While the details of how ATAT's generic access levels will differ by CSP, the following general guidelines apply:
- Portfolio Access Levels
    -  portfolio_administrator: Root level "break-glass" administrator account for all resources & operators under this Portfolio.
- Application & Environment Access Levels
    - administrator: Has full access to all assets under this Application/Environment.
    - contributor: Has access to create, update or remove resources under this Application/Environment. Cannot manage operator access.
    - read_only: Has access to view resources, configurations or logs under this Application/Environment, but cannot make any changes or create new resources.

Reply:
Applied in ce088ae6ae75128ef686dd681ce3af401b9d505f